### DISCUSSION TITLE
Network stitching into a new network.

### DISCUSSION THREAD

**Discussion by kizzle4shaw** [PMEAL/OpenPNM#2918] [Q&A]
I have created two cubic networks with different pore and throat distribution, the networks are pn and pn2, I tried using the stitch function to combine the networks but apparently the combination cannot be casted into a new network. My objective is to merge these networks and name them as pn3 which will be a new network with two different properties. Is there a way to achieve this? Maybe I'm misssing something, can someone please enlighten me.

**Comment by jgostick** [comment]
pn2 will be attached to pn1.  If you want to create three networks you should create a copy of pn1, like

``` python
from copy import deepcopy
pn3 = deepcopy(pn1)
op.topotools.stitch(pn3, pn2, ...)
```

**Reply by kizzle4shaw** [reply]
Perhaps I wasn't very clear with my question. After creation of pn1 and pn2, I want to combine them together to create pn3 which means pn3 will be a combination of the networks created before. Copying pn1 to pn3 wont change anything but the network name. Does the stitching function serve more purpose than just visualization?

Even in the workflow you have illustrated it shows to stitch them together, but where can the stitching output be applied?

**Reply by jgostick** [reply]
I thought you wanted `pn1` to remain unaffected by the stitch.  The call to stitch function will update `pn1` "in-place", meaning that `pn1` will have `pn2` attached to it.  The function doesn't return anything because it just updates `pn1` internally, ie. in-place.

**Reply by kizzle4shaw** [reply]
Thank you for the reply I have seen the changes. One last question if I may, is there a way to export the network into statoil format?
